Makefile

```diff
@@ -103,7 +103,7 @@ testshell: my.env .docker-build
 
 .PHONY: run
 run: my.env
-	${DC} up processor webapp webpack oidcprovider
+	${DC} up processor webapp webpack
```

Comment:
The `oidcprovider` service is a dependency of the `webapp`, so we don't need to run it here.